components/authz-service/storage/postgres/datamigration/sql/09_make_projects_wildcard_an_actual_project.up.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+INSERT INTO iam_projects (id, name, type) VALUES ('*', 'Wildcard project', 'chef-managed');
```

Comment:
we do have the ~~ALL PROJECTS~~ project . . . cannot recall why we didn't use `*` all the way down . . .

Reply:
(I'm not clear on why we need this migration)

Reply:
Frankly, I've made @msorens' approach work. I don't know! 💦

Reply:
Er, something is amiss 🤔

Reply:
https://github.com/chef/automate/pull/886/files#r304990794 maybe?

Reply:
I'm also confused. [This migration](https://github.com/chef/automate/blob/78ca2f0114d92ccdba09bc2c627fa9afd010460d/components/authz-service/storage/postgres/datamigration/sql/04_v2_set_projects_default.up.sql#L12) makes me think we're already populating empty statements with `~~All Projects~~`